grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -346,6 +356,16 @@ export const IntegrationForm = observer(
                 <div className={styles.webhookSwitch}>
                   <Switch value={field.value} onChange={field.onChange} />
                   <Text type="primary"> Create default outgoing webhooks</Text>
+                  <Tooltip
+                    content={
+                      <>
+                        If enabled, all the necessary webhooks will be created automatically. It's highly recommended to
+                        keep this enabled.
```

Comment:
Up to you but seems more appropriate in an enterprise context to refer exactly to **this option**
`It's highly recommended to keep this option enabled.`

Reply:
https://github.com/grafana/oncall/pull/4229/commits/41f1cba784fa9217e53a8a2269ebd6a3922f1ef2